### PR TITLE
Updated code and NPM package mentions

### LINKED ISSUE #873: Site: Update code & NPM references
* Code snippets (e.g. `<ThemeProvider>` --> `<SaltProvider>`)
* NPM package names (`@jpmorganchase/uitk-...` --> `@salt-ds/...`)
* NPM package version (we're going with 1.0.0!)

### PR DESCRIPTION
**Pull request by james-nash**
Closes #873

### CODE REVIEW

File: site/docs/getting-started/develop.mdx [hunk 1]
```diff
@@ -3,10 +3,6 @@ sidebar_label: Develop
 hide_table_of_contents: true
 ---
 
-import Link from "@docusaurus/Link";
```
Comment (james-nash) [inline site/docs/getting-started/develop.mdx:6]: FYI: These were redundant imports, so I took this opportunity to strip them out (I suspect VSCode auto-complete may have been overzealous and added them at some point due to the _example code_ on this page).

Comment (JoanaMMoreira) [reply site/docs/getting-started/develop.mdx:6]: I think this is required for the link on line 152

**Review by Fercas123 — DISMISSED** [review]

File: site/docs/getting-started/develop.mdx (hunk 1 above)
Comment (joshwooding) [reply site/docs/getting-started/develop.mdx:6]: I think so too

Comment (james-nash) [reply site/docs/getting-started/develop.mdx:6]: Oops. Looks like I got a bit trigger happy. Just pushed a fix for this

**Review by JoanaMMoreira — APPROVED** [review]

**Pull request merged**